Name: Sign In As Invisible [IM]
Version: 1.3.5-1
Author: Fritos1406
Description: Adds the ability/option to sign in as invisible

--- .orig/usr/palm/applications/com.palm.app.messaging/app/controllers/listview-assistant.js
+++ /usr/palm/applications/com.palm.app.messaging/app/controllers/listview-assistant.js
@@ -75,7 +75,7 @@
 			{label: $L('Available'), command:this.Messaging.Availability.getAvailabilityAsConstText(this.Messaging.Availability.AVAILABLE), secondaryIcon:'status-available'},
 			{label: $L('Busy'),      command:this.Messaging.Availability.getAvailabilityAsConstText(this.Messaging.Availability.BUSY),      secondaryIcon:'status-away'},
 			// libpurple doesn't support invisible status for gtalk so commenting this out for now.
-			//{label: $L('Invisible'), command:this.Messaging.Availability.getAvailabilityAsConstText(this.Messaging.Availability.INVISIBLE),  secondaryIcon:'status-invisible'},
+			{label: $L('Invisible'), command:this.Messaging.Availability.getAvailabilityAsConstText(this.Messaging.Availability.INVISIBLE),  secondaryIcon:'status-invisible'},
 			{label: $L('Sign off'),  command:this.Messaging.Availability.getAvailabilityAsConstText(this.Messaging.Availability.OFFLINE),   secondaryIcon:'status-offline'}
 		];
 		
@@ -1132,7 +1132,8 @@
 		
 		if (selectedValue != this.Messaging.Availability.OFFLINE &&
 		this.currentAvailability != this.Messaging.Availability.AVAILABLE &&
-		this.currentAvailability != this.Messaging.Availability.BUSY) {
+	    this.currentAvailability != this.Messaging.Availability.BUSY &&
+    	this.currentAvailability != this.Messaging.Availability.INVISIBLE) {
 			this.setAvailabilityIndicator(this.Messaging.Availability.PENDING);
 		} else {
 			this.setAvailabilityIndicator(selectedValue);
